RevenueCatUI/Templates/V2/Components/Stack/StackComponentViewModel.swift

```diff
@@ -136,39 +136,24 @@ struct StackComponentStyle {
             return .normal
         }
 
-        // Normal strategy for fit
-        switch self.size.height {
-        case .fit:
+        switch distribution {
+        case .start, .center, .end:
             return .normal
-        case .fill, .fixed:
-            break
-        }
-
-        // Normal strategy if start
-        guard case .start = distribution else {
+        case .spaceBetween, .spaceAround, .spaceEvenly:
             return .flex
         }
-
-        // WIP: Look deeper in tree
-//        if self.components.count > 3 {
-//            return .lazy
-//        } else {
-//            return .normal
-//        }
-        return .lazy
     }
 
     var hstackStrategy: StackStrategy {
         // Ensure horizontal
-        guard case .horizontal = self.dimension else {
+        guard case let .horizontal(_, distribution) = self.dimension else {
             return .normal
         }
 
-        // Not strategy for fit
-        switch self.size.width {
-        case .fit:
+        switch distribution {
+        case .start, .center, .end:
             return .normal
-        case .fill, .fixed:
+        case .spaceBetween, .spaceAround, .spaceEvenly:
             return .flex
```

Comment:
Flex stacks support `.start, .center, .end` distribution as well, so we could perfectly remove usages of VStack/HStack and leave just FlexVStack/FlexHStack.

However, the view structure of the latter are a bit more complex/nested than the simple stacks, so I've opted to keep them for now.